example/cloud-pubsub/subscribeMessage.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * This sample demonstrates how to subscribe from subscription.
+ */
+const main = async (topic = 'busTest') => {
+  const { CloudPubSubBus } = require('../../lib/cloud-pubsub-bus');
+
+  const clientConfig = { projectId: 'project-test', apiEndpoint: 'localhost:8085' };
+
+  const gcpOptions = {
+    clientConfig,
+    topicPrefix: 'topic-',
+    subscriptionPrefix: 'sub-',
+  };
+
+  // Create CloudPubSub Instance
+  const cloudPubSubBus = new CloudPubSubBus(gcpOptions);
+
+  // subscriptionName = subscriptionPrefix + topic
+  console.log(`Subscriber to subscription ${gcpOptions?.subscriptionPrefix + topic} is ready to receive messages`);
```

Comment:
gcpOption 이 nullable 일 필요는 없을 듯 해요.
```suggestion
  console.log(`Subscriber to subscription ${gcpOptions.subscriptionPrefix}${topic} is ready to receive messages`);
```
인터폴레이션 쓰는거 이렇게 해도...